src/test/java/bio/terra/stairway/FlightCleanerTest.java

```diff
@@ -0,0 +1,69 @@
+package bio.terra.stairway;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.stairway.fixtures.TestUtil;
+import bio.terra.stairway.flights.TestMinimalFlight;
+import java.time.Duration;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The intermediate states in this test may not be deterministic. All we can be sure of is that at
+ * the end we should have no flights in the flight list.
+ */
+@Tag("unit")
+public class FlightCleanerTest {
+  private static final Logger logger = LoggerFactory.getLogger(FlightCleanerTest.class);
+
+  @Test
+  public void successTest() throws Exception {
+    final String stairwayName = "flightCleanerTest";
+
+    // Start with a clean and shiny database environment.
+    Stairway stairway =
+        Stairway.newBuilder()
+            .stairwayName(stairwayName)
+            .completedFlightRetention(Duration.ofMinutes(1))
+            .retentionCheckInterval(Duration.ofSeconds(15))
```

Comment:
could we make these even shorter to let the test complete faster?

Reply:
Good idea. I separated the numbers out into constants and shrunk the time a bit.